### DISCUSSION TITLE
Remove post macro from api::core::emergency_access::post_emergency_access or add to routes!

### DISCUSSION THREAD

**Discussion by zacknewman** [dani-garcia/vaultwarden#4076] [Ideas]
[`api::core::emergency_access::post_emergency_access`](https://github.com/dani-garcia/vaultwarden/blob/main/src/api/core/emergency_access.rs#L101) is only ever used once in `put_emergency_access`, and the `Route` is not added by `routes!` in `routes`. Three fixes are:
* add the `Route` via `routes!` in `routes`.
* Remove the `post` macro.
* Remove the function altogether and integrate its body into `put_emergency_access`.

**Comment by BlackDex** [comment]
Good find. I'll take a look at it 😄
